authorize/run.go

```diff
@@ -151,41 +137,36 @@ func (a *Authorize) runDataTypeSyncer(ctx context.Context, typeURL string, updat
 			}
 
 			backoff.Reset()
-			serverVersion = res.GetServerVersion()
+			if res.GetServerVersion() != serverVersion {
+				serverVersion = res.GetServerVersion()
+				recordVersion = ""
```

Comment:
we weren't resetting this before. I wonder if that was causing problems. (though with the force sync I would've expected it to pick them up anyway)